patientsearch/src/js/components/Search.js

```diff
@@ -140,24 +164,38 @@ async function fetchData(url) {
     }
 
     return json;
-  }
+}
 
+function Alert(props) {
+    return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+  
 export default function Search() {
     let focusInput = React.useRef(null);
     const classes = useStyles();
     const [loading, setLoading] = React.useState(false);
     const [errorMessage, setErrorMessage] = React.useState("");
+    const [popOpen, setPop] = React.useState(false);
+    const [resultOpen, setResultOpen] = React.useState(false);
     const [firstName, setFirstName] = React.useState("");
     const [lastName, setLastName] = React.useState("");
     const [dob, setDOB] = React.useState(null);
     const [success, setSuccess] = React.useState(false);
     const buttonClassname = clsx({
         [classes.buttonSuccess]: success,
     });
+    const [searchResults, setSearchResults] = React.useState([]);
 
     //TODO this is not a correct one!
-    const getLaunchURL = () => {
-        return "https://cosri-dev.cirg.washington.edu/auth/launch?launch=eyJhIjoiMSIsImIiOiI0MTcwMiIsImUiOiJTTUFSVC0xMjM0In0&iss=https%3A%2F%2Fsmart-dev-sandbox-launcher.cirg.washington.edu%2Fv%2Fr2%2Ffhir";
+    const getLaunchURL = (patientId) => {
+        //TODO ALL OF THESE, hardcoded
+        patientId = patientId || 1102;
+        let baseURL = "https://backend.cosri-demo.cirg.washington.edu";
+        let iss = "https://fhir.cosri-demo.cirg.washington.edu/hapi-fhir-jpaserver/fhir/";
+        let launchParams = "patient=" + patientId;
+
+        return `${baseURL}/auth/launch?launch=${launchParams}&iss=${iss}`; 
```

Comment:
```suggestion
        return `${baseURL}/auth/launch?launch=${launchParam}&iss=${iss}`; 
```

Reply:
@ivan-c Ivan, I am still not seeing **patient id** in the search result.  Looking at the code, I realized that it is still a TODO based on what Paul has:  https://github.com/uwcirg/cosri-patientsearch/blob/df976726c8e113c3aa3f4e44b76712e82b9f8686/patientsearch/api.py#L172

Reply:
Nice find! I was just looking into that and [should have a fix here](https://github.com/uwcirg/cosri-patientsearch/pull/8), but shouldn't require any changes on your end.